OpenRA.Mods.Common/Traits/World/AutoSave.cs

```diff
@@ -0,0 +1,78 @@
+#region Copyright & License Information
+/*
+ * Copyright (c) The OpenRA Developers and Contributors
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[TraitLocation(SystemActors.World)]
+	[Desc("This trait allows setting a auto save options. Attach this to the World actor.")]
```

Comment:
```suggestion
	[Desc("This trait provides auto-save feature. Attach this to the World actor.")]
```

or

```suggestion
	[Desc("This trait makes auto-save work. Attach this to the World actor.")]
```

Not sure about `autosave`, `auto-save` or `auto save`, though.

Reply:
```suggestion
	[Desc("Add this trait to the world actor to enable auto-save.")]
```

perhaps